mobile/src/components/courses/components/course-card/components/vendor-label/vendor-label.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+
+import { View } from '~/components/common/common';
+import { vendorsKeyToImage } from '~/components/courses/components/course-card/common/map/maps';
+
+import { styles } from './style';
+
+type Props = {
+  vendorName: string;
```

Comment:
We should name images by vendor key
```suggestion
  vendorKey: string;
```